### PR TITLE
Packagified ProofPointDecodeURL

### PR DESCRIPTION
**Pull request by guykeller**
## Status 
Ready

## Description
Packagified ProofPointDecodeURL

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

File: Tests/scripts/pkg_dev_test_tasks.py [hunk 1]
```diff
@@ -63,8 +61,10 @@ def get_python_version(docker_image):
     print("Detected python version: [{}] for docker image: {}".format(py_ver, docker_image))
     py_num = float(py_ver)
     if py_num < 2.7 or (3 < py_num < 3.4):  # pylint can only work on python 3.4 and up
-        raise ValueError("Python version for docker image: {} is not supported: {}. "
-                         "We only support python 2.7.* and python3 >= 3.4.".format(docker_image, py_num))
+        print_warning("Python version for docker image: {} is not supported: {}. "
```
Comment (glicht) [inline Tests/scripts/pkg_dev_test_tasks.py]: Don't change this. We don't support python 3.4. In such cases we need to change the python docker image.

File: Scripts/ProofpointDecodeURL/ProofpointDecodeURL.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+commonfields:
+  id: ProofpointDecodeURL
+  version: -1
+name: ProofpointDecodeURL
+script: ''
+type: python
+subtype: python3
```
Comment (glicht) [inline Scripts/ProofpointDecodeURL/ProofpointDecodeURL.yml]: change to: python2

Comment (guykeller) [reply Scripts/ProofpointDecodeURL/ProofpointDecodeURL.yml]: fixed

File: Tests/scripts/pkg_dev_test_tasks.py [hunk 3]
```diff
@@ -64,7 +64,8 @@ def get_python_version(docker_image):
     py_num = float(py_ver)
     if py_num < 2.7 or (3 < py_num < 3.4):  # pylint can only work on python 3.4 and up
         raise ValueError("Python version for docker image: {} is not supported: {}. "
-                         "We only support python 2.7.* and python3 >= 3.4.".format(docker_image, py_num))
+                      "We only support python 2.7.* and python3 >= 3.4.".format(docker_image, py_num))
```
Comment (glicht) [inline Tests/scripts/pkg_dev_test_tasks.py]: Not sure why you have this change. Flake8 doesn't like it.

File: Scripts/ProofpointDecodeURL/CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+## [Unreleased]
+-
```
Comment (glicht) [inline Scripts/ProofpointDecodeURL/CHANGELOG.md]:
```suggestion
Deprecated. Changed to call *UnEscapeURLs*.
```

File: Scripts/ProofpointDecodeURL/ProofpointDecodeURL.yml [hunk 5]
```diff
@@ -0,0 +1,22 @@
+commonfields:
+  id: ProofpointDecodeURL
+  version: -1
+name: ProofpointDecodeURL
+script: ''
+type: python
+subtype: python2
+tags: []
+comment: Decode ProofPoint URLs to get the actual URLs
```
Comment (glicht) [inline Scripts/ProofpointDecodeURL/ProofpointDecodeURL.yml]:
```suggestion
comment: Deprecated. Use UnEscapeURLs instead. Decode ProofPoint URLs to get the actual URLs.
```

**Pull request merged**